src/components/notes/list.jsx

```diff
@@ -1,116 +1,75 @@
-        <Item {...props} />
-      </th>
-      <td className="c-table-cell">
-        <time datatime={props.note.cozyMetadata.updatedAt}>
-          {formatedUpdatedAt}
-        </time>
-      </td>
-      <td className="c-table-cell">—</td>
-      <td className="c-table-cell">
-        <Button
-          theme="action"
-          extension="narrow"
-          icon={<Icon icon="dots" color="coolGrey" width="17" height="17" />}
-          iconOnly
-          label="actions"
-        />
-      </td>
-    </tr>
-  )
-}
-
-const List = props => {
-  const { notes } = props
-  return !notes || !notes.length ? null : (
-    <table className="notes-list c-table">
-      <thead className="c-table-head">
-        <tr className="c-table-row-head">
-          <th className="c-table-header">Nom</th>
-          <th className="c-table-header">Dernière mise à jour</th>
-          <th className="c-table-header">Partages</th>
-          <th className="c-table-header" />
-        </tr>
-      </thead>
-      <tbody>
-        {notes.map(note => (
-          <Row key={note._id} note={note} />
-        ))}
-      </tbody>
-    </table>
-  )
-}
-
-const ConnectedList = props => {
-  const { data, fetchStatus } = props.notes
-  // cozy-client statuses
-  const isLoading = fetchStatus === 'loading' || fetchStatus === 'pending'
-  return (
-    <div className="notes notes-list-container">
-      {isLoading ? (
-        <Spinner size="xxlarge" middle />
-      ) : (
-        <div>
-          <ListHeader />
-          <List notes={data} />
-        </div>
-      )}
-    </div>
-  )
-}
-
-*/
-
-const ListHeader = () => {
-  return (
-    <>
-      <HeaderMenu
-        left={<MainTitle tag="h1">Mes notes</MainTitle>}
-        right={<Add />}
-      />
-    </>
-  )
-}
-
-export default ListHeader
+))
+/* const cellStyles = { flexGrow: 1 }
+const nameCellStyles = { flexGrow: 10 } */
+const List = translate()(({ t }) => (
+  <Table>
+    <TableHead>
+      <TableRow>
+        <TableHeader className="tableCellName">
+          {t('Notes.List.name')}
+        </TableHeader>
+        <TableHeader className="tableCell">
+          {t('Notes.List.updated_at')}
+        </TableHeader>
+        <TableHeader className="tableCell">
+          {t('Notes.List.location')}
+        </TableHeader>
+        <TableHeader className="tableCell">
+          {t('Notes.List.sharings')}
+        </TableHeader>
+      </TableRow>
+    </TableHead>
+    <TableBody>
+      <TableRow className="tableRowEmpty">
+        <TableCell className="tableCellEmpty">
+          <EmptyComponent />
+        </TableCell>
+      </TableRow>
+    </TableBody>
+  </Table>
+))
+
+const TitleApp = translate()(({ t }) => (
+  <MainTitle className="u-pl-1">{t('Notes.List.my_notes')}</MainTitle>
+))
+
+const EmptyPage = withBreakpoints()(({ breakpoints: { isMobile } }) => (
+  //On Mobile, we don't want a margin top since we don't have title and
+  //we want the component to start at the end of the bar
```

Comment:
The comment is confusing me, why do we not have a title on mobile? `<TitleApp />` is right there and I don't see a condition hiding it on mobile?

Reply:
TitleApp is not displayed on Mobile. You right, this `mt` should be on TitleApp and not on Stack

Reply:
fixed